layouts/fastly/helix.vcl

```diff
@@ -28,7 +28,7 @@ sub hlx_strain {
     set req.http.X-Strain = "default";
 
     # run custom strain resolution
-    include "strains.vcl";
+    include "dynamic.vcl";
```

Comment:
I would probably split the two includes. Keep one `strains.vcl` and have an entirely new `dynamic.vcl`